docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,188 @@
+
+##  Installation & Configuration
+
+* **Convenience build**
+
+
+* **Instance.env** - V1 Only
+
+  The Zowe instance directory contains a file instance.env that stores the Zowe configuration data. The data is read each time Zowe is started. You can modify instance.env to configure the Zowe runtime. See https://docs.zowe.org/stable/user-guide/configure-instance-directory/#updating-the-instanceenv-configuration-file 
+
+* **Zowe runtime**
+ 
+* **Started task** 
+
+  a started task (STC) is a type of runnable/running program on z/os. Zowe started task (See also ZWESVSTC) – Does it need to go in Glossary? https://www.ibm.com/docs/en/zos/2.1.0?topic=tasks-determining-whether-use-started-task
+  Zowe has 3 actually:
+  - ZWESVSTC: v1 primary server STC
+  - ZWESLSTC: v1 HA/FT primary server STC, or the primary server STC for v2 (SVSTC goes away in v2)
+  - ZWESISTC: ZIS STC
+
+* **workspace directory**
+
+* **Zowe configuration file**
+
+  Zowe.yaml: It is a YAML file that is required to configure Zowe runtime. It can be used as an alternative to instance.env if you have to configure Zowe in more granular level. It is also required to start Zowe in high availability mode. See https://docs.zowe.org/stable/user-guide/configure-instance-directory/#creating-the-zoweyaml-file 
+
+* **Zowe instance**
+    
+* **Zowe instance directory** 
+
+  Also known as <INSTANCE_DIR>. It contains information that is specific to a launch of Zowe. It contains configuration settings that determine how an instance of the Zowe server is started, such as ports that are used or paths to dependent Java and Node.js runtimes.  The instance directory also contains log directory where different 'microservices' write trace data for diagnosis, as well as a workspace and shell scripts to start and stop Zowe.
+
+* **Zowe sample library**
+
+  (See also SZWESAMP) – Does it need to go in Glossary?
+
+* **ZWESVUSR** 
+
+  This is a started task ID used to run the PROCLIB ZWESVSTC. The task starts a USS environment using BPXBATSL that executes the core Zowe Desktop (ZLUX) node.js server, the Java API Mediation Layer, and the Z Secure Services C component. To work with USS, the user ID ZWESVUSR must have a valid OMVS segment. For more information, see System requirements | Zowe Docs
+
+* **Zowe runtime directory**
+
+  referred to as `<RUNTIME_DIR>`. Also known as ROOT DIR is the directory where the Zowe runtime is located, also referred to as the <RUNTIME_DIR>. Defaults to the location of where zowe-configure-instance was executed. See Creating and configuring the Zowe instance directory | Zowe Docs 
+
+## Operation
+
+* **App2App**
+
+  It is a unique feature to Zowe environment where one application plug-in can communicate with another and it is applicable to multiple application plug-ins. The application framework provides constructs that facilitate this ability. See Application-to-application communication | Zowe DocsConfig Service: 
```

Comment:
Looks like "config service" was at the end here.
https://docs.zowe.org/stable/extend/extend-desktop/mvd-configdataservice/

The Config(uration) Service is a part of the Application Framework which allows plugins and the framework itself to store user configuration as JSON or binary formats. The configuration is stored in a hierarchy in which company-wide and system-wide defaults can exist for all users, and users may override the defaults if policy allows it. What can be stored and what can be overridden is according to plugin definition and administrative configuration.